code/modules/paperwork/paperbin.dm

```diff
@@ -36,9 +50,23 @@
 		papers = null
 	. = ..()
 
+/obj/item/paper_bin/dump_contents(atom/droppoint, collapse = FALSE)
+	if(!droppoint)
+		droppoint = drop_location()
+	if(collapse)
+		visible_message("<span class='warning'>The stack of paper collapses!</span>")
+	for(var/atom/movable/AM in contents)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names